### DISCUSSION TITLE
NOT WORKING ON WINDOWS 7

### DISCUSSION THREAD

**Discussion by Rainman69** [KRTirtho/spotube#697] [Q&A]
![Screenshot_1](https://github.com/KRTirtho/spotube/assets/96986338/1627694e-2978-4d11-a097-10ad8d6f978e)

**Comment by KRTirtho** [comment]
Only Windows 10 & 11 is supported. Windows 8 status unknown. Windows 7 completely doesn't work

**Reply by Rainman69** [reply]
well
